source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,286 @@
+.. _csharp-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query to match a subset
+of documents using the {+driver-long+}.
+
+Most CRUD operations allow you to narrow the set of matched documents
+by passing an expression as an argument to the method, or by specifying
+matching criteria in a **query filter**. Query filters contain one or more query
+operators to determine which documents to include in the resulting set.
```

Comment:
- Added CRUD acronym meaning.

- The wording here says "most CRUD operations" since you don't do any matching on documents with create operations. As far as I know all others do let you.

- I changed the first instance of method to say "driver method" to help make the distinction more clear. Let me know if you have other suggestions.

- I restructured the paragraph to better define a query filter in the way that it is now used throughout the page.

- Removed for new definition